keg-tester

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python3
+
```

Comment:
I assume this part will go into the keg repo and into a "real" implementation ?

Reply:
Yes, see initial comment.